Minio/DataModel/BucketOperationsArgs.cs

```diff
@@ -430,6 +430,9 @@ public override RestRequest BuildRequest(RestRequest request)
             request.AddQueryParameter("tagging","");
             string body = this.BucketTags.MarshalXML();
             request.AddParameter(new Parameter("text/xml", body, ParameterType.RequestBody));
+            request.AddOrUpdateParameter("Content-MD5",
```

Comment:
is this computed for all operations?

Reply:
All operations with XML configuration attached, we get an exception if 'Content-MD5' is missing